task3_snkrs/spiders/snkrs_spider.py

```diff
@@ -0,0 +1,86 @@
+import scrapy
+from task3_snkrs.items import SnkrsItem
+
+
+class SnkrsSpider(scrapy.Spider):
+    name = 'snkrs'
+    response = None
```

Comment:
Scrapy requests works asynchronously we cannot  be sure its value will not change until item yields.